const_linux.go

```diff
@@ -0,0 +1,4 @@
+package main
+
+// MaxCommandBytes is the maximum number of bytes supported by exec
+const MaxCommandBytes = 2097152
```

Comment:
I'm not sure this is (always) correct. For example:

```
$ grep ARG_MAX /usr/include/linux/limits.h 
#define ARG_MAX       131072    /* # bytes of args + environ for exec() */
```

This also (as the comment suggests) includes environment space, so in practice it will be lower. I think it is probably good to be conservative with these constants. Perhaps 32767 for all?